README.md

```diff
@@ -23,13 +23,13 @@ python src/download_data.py download_data.py --url=https://github.com/invinst/ch
 # Clean / preprocess data
 Rscript src/read_preprocess_data.R --file_path=data --out_dir=data/processed
 
-# create exploritory data analysis figures and write to file
+# Create exploritory data analysis figures and write to file
 Rscript src/generate_EDA_figures.R --all_data=data/complaints_all_staff --police_data=data/complaints_police --out_dir=eda/images
```

Comment:
The script is flexible, just wondering if we needed the actual code to run or if this is a sample. The below (I think) should be the actual code, since I refer to the csv as inputs. Updated the location as well for the preprocessed folder

`Rscript generate_EDA_figures.r --all_data=../data/processed/complaints_all_staff.csv --police_data=../data/processed/complaints_police.csv --out_dir=../eda/images
`